From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jordan Christiansen <xordspar0@gmail.com>
Date: Sun, 15 Mar 2020 16:55:33 -0500
Subject: [PATCH] Remove redundant demoextend definition

GCC 10 enables -fno-common by default, which causes the linker to fail when
there are multple definitions of a global variable.

See https://gcc.gnu.org/gcc-10/porting_to.html
---
 src/hexen/mn_menu.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/hexen/mn_menu.c b/src/hexen/mn_menu.c
index a3ff999801b657a6dca4d1e62489d65bafe4e091..b9ad45eea7c915a4066edabff1c12cead2a6ae76 100644
--- a/src/hexen/mn_menu.c
+++ b/src/hexen/mn_menu.c
@@ -131,7 +131,6 @@ boolean MenuActive;
 int InfoType;
 int messageson = true;
 boolean mn_SuicideConsole;
-boolean demoextend; // from h2def.h
 
 // PRIVATE DATA DEFINITIONS ------------------------------------------------
 
